encryption/handshaker_mock.go

```diff
@@ -0,0 +1,72 @@
+package encryption
```

Comment:
It looks like you just ignored my comments in the previous review.
If you use this mock just for testing, on your local environment, that's fine, but you can't commit this implementation.